deployment/debian-package-x64/pkg-src/conf/jellyfin

```diff
@@ -19,13 +19,17 @@ JELLYFIN_LOG_DIRECTORY="/var/log/jellyfin"
 JELLYFIN_CACHE_DIRECTORY="/var/cache/jellyfin"
 
 # Restart script for in-app server control
-JELLYFIN_RESTART_OPT="--restartpath /usr/lib/jellyfin/restart.sh"
+JELLYFIN_RESTART_OPT="--restartpath=/usr/lib/jellyfin/restart.sh"
 
-# [OPTIONAL] ffmpeg binary paths
-#JELLYFIN_FFMPEG_OPTS="--ffmpeg /usr/bin/ffmpeg --ffprobe /usr/bin/ffprobe"
+# [OPTIONAL] ffmpeg binary paths, overriding the UI-configured values
+#JELLYFIN_FFMPEG_OPT="--ffmpeg=/usr/bin/ffmpeg"
+#JELLYFIN_FFPROBE_OPT="--ffprobe=/usr/bin/ffprobe"
 
-# [OPTIONAL] Additional user-defined options for the binary
-#JELLYFIN_ADD_OPTS=""
+# [OPTIONAL] run Jellyfin as a headless service
+#JELLYFIN_SERVICE_OPT="--service"
+
+# [OPTIONAL] run Jellyfin without the web app
+#JELLYFIN_NOWEBAPP_OPT="--noautorunwebapp"
```

Comment:
Does this option make any sense on Linux? What does it do anyway?

Reply:
No idea, but as a flag available I added it.